perun-db/postgres.sql

```diff
@@ -1550,6 +1550,58 @@ create table groups_to_register (
     constraint grpreg_group_fk foreign key (group_id) references groups(id) on delete cascade
 );
 
+-- CONSENTS_DEFINITIONS - consents definitions for facilities, services and attributes
+create table consents_definitions (
+	id integer not null,
+	name varchar not null,
+	decription varchar,
+	created_at timestamp default statement_timestamp() not null,
+	created_by varchar default user not null,
+	modified_at timestamp default statement_timestamp() not null,
+	modified_by varchar default user not null,
+	created_by_uid integer,
+	modified_by_uid integer,
+	constraint con_def_pk primary key(id)
```

Comment:
I think we should also add constraint for the name to be unique, what do you think?